public/assets/js/publishFunctions.js

```diff
@@ -22,28 +22,28 @@ function validateFile(file) {
 			}
 			break;
 		default:
-			throw new Error('The ' + file.Type + ' content type is not supported.  Only, .jpeg, .png, .gif, and .mp4 files are currently supported.')
+			throw new Error(file.type + ' is not supported a supported file type. Only, .jpeg, .png, .gif, and .mp4 files are currently supported.')
 	}
+	// validate the file name (note: different from the lbry claim name)
+	var invalidCharacter = /[^\w.-\s()]/g.exec(file.name);
+	if (invalidCharacter) {
+		throw new Error('Special characters, such as "' + invalidCharacter + '", are not allowed in the file name.');
+	};
 }
-
-function validateSubmission(stagedFiles, name){
-	// make sure only 1 file was selected
-	if (!stagedFiles) {
-		throw new FileError("Please select a file");
-	} else if (stagedFiles.length > 1) {
-		throw new FileError("Only one file is allowed at a time");
-	}
-	// validate 'name' field
-	var invalidCharacters = /[^A-Za-z0-9,-]/.exec(name);
-	if (invalidCharacters) {
-		throw new NameError(invalidCharacters + ' is not allowed. A-Z, a-z, 0-9, and "-" only.');
-	} else if (name.length < 1) {
-		throw new NameError("You must enter a name for your claim");
-	}
-}
-
+// validation function that checks to make sure the claim name is not already claimed
 function validateClaimName (name) {
```

Comment:
@98farhan94 note that there are two functions in this file.  One to validate the file itself, and one to validate the claim name that the user chooses.  The file name can have spaces in it, but the claim name cannot.